src/jumpstart/useAddAssetsActions.tsx

```diff
@@ -0,0 +1,60 @@
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+import AppAnalytics from 'src/analytics/AppAnalytics'
+import { JumpstartEvents } from 'src/analytics/Events'
+import { type AddAssetsAction } from 'src/components/AddAssetsBottomSheet'
+import { CICOFlow, FiatExchangeFlow } from 'src/fiatExchanges/utils'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { isAppSwapsEnabledSelector } from 'src/navigator/selectors'
+import { TokenActionName } from 'src/tokens/types'
+import networkConfig from 'src/web3/networkConfig'
+
+export function useAddAssetsActions(): AddAssetsAction[] {
```

Comment:
i don't have strong feelings about this abstraction but i do wonder if it's necessary 😅 it's used in one place, why not keep it in the intro screen?

Reply:
@kathaypacific Definitely not necessary! That's just my habit of decomposing components into smaller pieces when I feel like the component is too bloated but that's probably a "me" issue 😄